File: packages/engine/test/fixtures/python/hello_world.py
"""Greeting demo."""

GREETING = "Hello"


class Greeter:
    """Greets people."""

    prefix: str = "Hi"

    def __init__(self, prefix: str):
        self.prefix = prefix

    def greet(self, name: str) -> str:
        return f"{self.prefix}, {name}"


def greet_user(name: str) -> str:
    return f"{GREETING}, {name}"

File: packages/engine/test/fixtures/typescript/hello_world.ts
/** Greeting demo. */

export const GREETING = "Hello";

export class Greeter {
  /** Greets people. */
  prefix: string = "Hi";

  constructor(prefix: string) {
    this.prefix = prefix;
  }

  greet(name: string): string {
    console.log(name);
    return `${this.prefix}, ${name}`;
  }
}

export function greetUser(name: string): string {
  return `${GREETING}, ${name}`;
}
